scripts/update_version.py

```diff
@@ -42,7 +40,7 @@ def update_version():
 
     # docs/source/_static/versions.json for readthedocs build
     with open(
-        os.path.join(pybamm.root_dir(), "docs", "source", "_static", "versions.json"),
+        os.path.join(pybamm.root_dir(), "docs", "_static", "versions.json"),
```

Comment:
In one of the previous meetings @tinosulzer and I had decided not to enable `rcX` documentation builds and keep them for just stable releases. Could we add a check here too, with a conditional `if "rc" in release_version`? Also, a suggestion: I think splitting all the file-handling operations per file into their own separate functions would be great for improving readability

Reply:
Oh yes, I remember that, thanks!